### PR TITLE
[DEV-3140] Program switch

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> Program Switch required for rendering different program widgets within a global widget
> Stencilbook components and stories added to bedrock components

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

 - Jira issue number: https://saasquatch.atlassian.net/browse/DEV-3140
 - Process.st launch checklist: https://app.process.st/runs/Bedrock-Components-Program-Switch-kdOXHbuNeHc7VXkGFOhOyQ

### Development

- [x] [Prettier](https://www.npmjs.com/package/prettier) was run
- [x] The behaviour changes in the pull request are covered by specs
- [x] All tests related to the changed code pass in development

### Paperwork

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has a Jira number
- [x] This pull request has a Process.st launch checklist
- [x] "Ready for review" label attached to the PR and reviewers pinged on Slack

### Code review 

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/bedrock-components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/bedrock-components",
-  "version": "1.1.1",
+  "version": "1.1.2-4",
```
Comment (00salmon) [inline packages/bedrock-components/package.json:3]:
```suggestion
  "version": "1.1.2",
```

File: packages/bedrock-components/src/index.html [hunk 2]
```diff
@@ -4,57 +4,38 @@
...
+      </sqm-program-menu>
+      <sqb-program-switch>
+        <template program-id="Vacay-referral">
+          <sqb-widget widget-type="p/Vacay-referral/w/referrerWidget"></sqb-widget>
+        </template>
+        <template program-id="vacay-affiliates">
+          <sqb-widget widget-type="p/vacay-affiliates/w/referrerWidget"></sqb-widget>
+        </template>
+      </sqb-program-switch>
+    </sqb-program-section> -->
```
Comment (00salmon) [inline packages/bedrock-components/src/index.html:39]:
```suggestion
    <sqb-stencilbook></sqb-stencilbook>
```

File: packages/bedrock-components/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saasquatch/bedrock-components",
-  "version": "1.1.1",
+  "version": "1.1.2-5",
```
Comment (00salmon) [inline packages/bedrock-components/package.json:3]:
```suggestion
  "version": "1.1.2",
```

File: packages/bedrock-components/CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.2] - 2021-10-27
+
+### Added
+
+- \<sqb-program-switch> - Loads different content based on the programId from useProgramId
```
Comment (DerekSiemens) [inline packages/bedrock-components/CHANGELOG.md:14]: Needs sqb-widget update here as well

File: packages/bedrock-components/src/components/sqb-widget/sqb-widget.feature [hunk 5]
```diff
@@ -1,10 +1,69 @@
...
+            }
+            """
+        Examples:
+            | mayHaveTrackLoads        | value | mayOccur      | mayHave |
+            | has track-loads          | true  | occurs        | has     |
+            | has track-loads          | false | doesn't occur | N/A     |
+            | doesn't have track-loads |       | doesn't occur | N/A     |
+
+    @motivating
+    Scenario Outline: The engagement medium of load events can be configured but defaults to `EMBED`
```
Comment (loganvolkers) [inline packages/bedrock-components/src/components/sqb-widget/sqb-widget.feature:50]: Cannot be configured
```suggestion
    Scenario Outline: The engagement medium of load events defaults to `EMBED`
```

**Review by loganvolkers — APPROVED** [review]
Looks good to me

**Pull request merged**